contracts/src/v0.1/CoordinatorBase.sol

```diff
@@ -226,4 +252,34 @@ abstract contract CoordinatorBase is Ownable, ICoordinatorBase {
             return false;
         }
     }
+
+    function serviceFeeByAcc(uint64 accId, uint32 numSubmission) internal returns (uint256) {
+        (, uint64 reqCount, , , IAccount.AccountType accType) = sPrepayment.getAccount(accId);
+        (uint256 sStartTime, uint256 sPeriod, , uint256 sSubscriptionPrice) = sPrepayment
+            .getAccountDetail(accId);
+
+        if (accType == IAccount.AccountType.FIAT_SUBSCRIPTION) {
+            sPrepayment.increaseReqCount(accId);
+            return 0;
+        } else {
+            if (accType == IAccount.AccountType.KLAY_SUBSCRIPTION) {
+                sPrepayment.increaseReqCount(accId);
+            }
+            uint256 serviceFee;
+            if (accType == IAccount.AccountType.KLAY_SUBSCRIPTION) {
+                uint256 index = (block.timestamp - sStartTime) / sPeriod;
+                if (!sPrepayment.getSubscriptionPaid(accId, index)) {
+                    serviceFee = sSubscriptionPrice;
+                    sPrepayment.updateSubscriptionPaid(accId, index, true);
+                } else {
+                    return 0;
+                }
+            } else if (accType == IAccount.AccountType.KLAY_DISCOUNT) {
+                serviceFee = (serviceFee * sPrepayment.getFeeRatio(accId)) / 10_000;
```

Comment:
The result of this `serviceFee` will be always zero, because the `serviceFee` on the right side has zero value.